modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/DecimalInput.es.js

```diff
@@ -33,9 +39,9 @@ class DecimalInput extends React.Component {
 	};
 
 	render() {
-		const {disabled, value} = this.props;
+		const {disabled, options, value} = this.props;
 
-		return (
+		return options.length === 0 ? (
```

Comment:
I think here it's not necessary `=== 0`. It should work with `options. length ? ... : ...`, if I'm not wrong. 🤔

Reply:
I'm not sure. I've followed the pattern here
https://github.com/liferay-tango/liferay-portal/blob/master/modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/StringInput.es.js#L38

Reply:
Oh, I see...We can let the condition as it is. Maybe we should refactor this in the whole module 👍  Thanks.

Reply:
FWIW, I usually recommend the `.length` check instead of `.length === 0` as it is more "idiomatic", but you will find both in liferay-portal.

I made [an issue](https://github.com/liferay/eslint-config-liferay/issues/191) to track adding a lint for this.

Reply:
Thanks Greg! I will update those checks because there are a few in the whole module :)